app/Http/Controllers/StudentsController.php

```diff
@@ -27,7 +27,8 @@ public function create(Family $family)
     {
         $families = Family::all();
         $family_id = $family->id;
-        return view('students.create', compact('family_id', 'families'));
+        $family_last = $family->Last;
```

Comment:
Why not just return the entire `$family` object in the view instead of extracting individual properties to their own variable?